packages/graphql-codegen-supermassive-typed-document-node-plugin/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@graphitation/graphql-codegen-supermassive-typed-document-node-plugin",
+  "license": "MIT",
+  "version": "0.0.1",
+  "main": "./src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/graphitation.git",
+    "directory": "packages/graphql-codegen-supermassive-typed-document-node-plugin"
+  },
+  "scripts": {
+    "build": "monorepo-scripts build --outDir='./'",
+    "lint": "monorepo-scripts lint",
+    "types": "monorepo-scripts types",
+    "just": "monorepo-scripts"
+  },
+  "devDependencies": {
+    "@graphitation/supermassive": "*",
+    "@graphql-codegen/plugin-helpers": "^1.18.2",
+    "@graphql-codegen/visitor-plugin-common": "^1.17.20",
+    "@types/jest": "^26.0.22",
+    "graphql-tag": "^2.11.0",
```

Comment:
Do we really need to use this package? We actually have one ourselves that does what we would normally need.

Reply:
Let’s follow up on this after your vacation. Can you file a work item?